ddocs/medic-client/views/contacts_by_type_index_name/map.js

```diff
@@ -1,10 +1,11 @@
 function(doc) {
   var types = [ 'district_hospital', 'health_center', 'clinic', 'person' ];
   var idx = types.indexOf(doc.type);
-  var dead = !!doc.date_of_death;
   if (idx !== -1) {
+    var dead = !!doc.date_of_death;
+    var muted = !!doc.muted;
     var name = doc.name && doc.name.toLowerCase();
-    var order = dead + ' ' + idx + ' ' + name;
+    var order = muted + ' ' + dead + ' ' + idx + ' ' + name;
```

Comment:
Maybe this has been discussed? But I probably would have sorted by death first and muted second? So all dead contacts are together at the actual bottom (completely ignored) and then alive+muted contacts are together slightly above that.

Reply:
Done! 

![image](https://user-images.githubusercontent.com/35681649/57636981-f36ac500-75b2-11e9-8090-48bc63a26d8a.png)